web_ui/src/pages/user-management/users/add-member-popup/add-member-popup.component.tsx

```diff
@@ -217,10 +227,62 @@ export const AddMemberPopup = ({ organizationId, workspaceId }: AddMemberPopupPr
                                     flex={1}
                                 />
                                 <RolePicker
-                                    roles={roles}
-                                    selectedRole={selectedRole}
-                                    setSelectedRole={setSelectedRole}
+                                    roles={orgRoles}
+                                    selectedRole={selectedOrgRole as USER_ROLE}
+                                    label='Organization Role'
+                                    setSelectedRole={(r) => {
```

Comment:
As a general rule, we avoid using single-letter variables; they don’t help with readability